presto-main/src/main/java/com/facebook/presto/security/FileBasedSystemAccessControl.java

```diff
@@ -78,14 +80,36 @@ public SystemAccessControl create(Map<String, String> config)
             checkState(
                     configFileName != null,
                     "Security configuration must contain the '%s' property", CONFIG_FILE_NAME);
-
-            try {
-                Path path = Paths.get(configFileName);
-                if (!path.isAbsolute()) {
-                    path = path.toAbsolutePath();
+            final Path path = Paths.get(configFileName);
+            return RefreshingSystemAccessControl.optionallyRefresh(config, new Supplier<Optional<SystemAccessControl>>()
```

Comment:
I think it would be nice to provide `RefreshingSystemAccessControlFactory` so it could be used on top of  regular. `FileBasedSystemAccessControl.Factory`

Reply:
Im not sure how that would help. In many cases it might be bad. This way people can intentionally add it with just a couple lines of code change and ensure it fits their model